ksqldb-execution/src/main/java/io/confluent/ksql/execution/transform/KsqlTransformer.java

```diff
@@ -29,5 +29,5 @@
  */
 public interface KsqlTransformer<K, R> {
```

Comment:
[FYI] None of the implementation is making use of context.

Implementations of KsqlTransformer  (7 usages found)
    - KudafAggregator.ResultTransformer
    - KudtfFlatMapper
    - SelectValueMapper.SelectMapper
    - SqlPredicate.Transformer
    - StreamAggregateBuilder.WindowBoundsPopulator
    - lambda in StreamSelectBuilder.build
    - lambda in TableSuppressBuilder.build